stylesheets/components/cards.css

```diff
@@ -0,0 +1,91 @@
+.card-container {
+  background-color: var(--white-color);
+  border: 1.33px solid #a3ce53;
+  border-radius: var(--border-rd-6);
+  width:18.5rem;
+  overflow: hidden;
+  /* box-shadow: 0px 0px 15px hsl(0, 0%, 80%); */
+}
+.card{
+    gap:var(--spacing-xs);
+    position: relative;
+}
+.card-img-container{
+    height:17rem;
+}
+.card-img{
+    width: 100%; 
+    max-height: 100%; 
+    /* margin: 0 auto; */
+}
+.wishlist-icon{
+    color:hsl(0, 0%, 75%);
+    position: absolute;
+    top:4px;
+    right:4px;
+    font-size: 0.2rem;
+}
+.rating-icon-wrapper{
+font-size: 2px;
+}
+.card-body{
+  margin-left:1rem;
+  margin-right:1rem;
+  margin-bottom: 1rem;
+  gap:0.2rem;
+}
+.card-text{
+    color:var(--text-color);
+    font-size: 0.9rem;
+    font-weight: 300;
+}
+.card-section{
+    justify-content: space-between;
+}
+.card-price{
+    gap:var(--spacing-xs);
+}
+.card-current-price{
+    font-size:1.2rem;
+    font-weight: 600;
+}
+.card-price-striked{
+    text-decoration: line-through;
+}
+.rating-icon-wrapper{
+    gap:0.1rem;
+}
+.rating-icon{
+    color:var(--yellow-dark);
+    font-size:12px;
+}
+.rating-count{
+    font-size: 0.85rem;
+}
+.card-actions{
+    margin-top:2rem;
+    gap:1rem;
+}
+.quantity{
+    border:1px solid hsl(0, 0%, 80%);
+    border-radius:3px;
+
+}
+.quantity p{
+    padding:0.3rem 0.6rem;
+}
+.add-to-cart{
+    gap:var(--spacing-xs);
+    background-color: #a3ce53;
```

Comment:
you could give value of color as varibale from utili class

Reply:
same goes here, I did change this but forgot to push